vrk-kpa/suomidigi
# SUOPA-516 Bookmarks

To test:
- `make fresh`
- Log in with any user and test out the bookmarks functionality
   - Start by going to these pages: 
      - https://suomidigi.docker.sh/kehittamisesimerkit
      - https://suomidigi.docker.sh/blogit
      - https://suomidigi.docker.sh/tapahtumat
   - Open few nodes per each page to a new tab and click on "Tallenna suosikkeihin"
   - After you've added enough nodes to bookmarks, head over to https://suomidigi.docker.sh/user and check that every node you've added to bookmarks has appeared to "Tallentamani suosikit" list.
